analyticsdataserver/router.py

```diff
@@ -1,15 +1,18 @@
 from django.conf import settings
 
+from analytics_data_api.middleware import thread_data
+
 
 class AnalyticsApiRouter:
     def db_for_read(self, model, **hints):  # pylint: disable=unused-argument
         # pylint: disable=protected-access
         return self._get_database(model._meta.app_label)
 
     def _get_database(self, app_label):
-        if app_label in ('v0', 'enterprise_data'):
+        if app_label in ('v0', 'v1', 'enterprise_data'):
```

Comment:
does this mean it could just look at app_label = v1 right here and not have to look at the thread_data?

Reply:
Unfortunately, I don't think so. My understanding is that we do not have any models with the `v1` app label. The app label will be the application that the model is defined in. All the models will have the `v0` label until we add models to the `v1` app.

I suppose this code change doesn't do anything, so I'm happy to remove it. I was being overly cautious in seeing this and thinking, "this may trip us up in the future if we add models to v1 and don't realize we need to change this."

Reply:
ah I think I understand

that sounds like good caution to me